flow.json

```diff
@@ -98,15 +107,19 @@
 				"ExampleNFT",
 				"ExampleToken",
 				"LostAndFound",
-				"FeeEstimator"
+				"FeeEstimator",
+				"LostAndFoundHelper"
 			],
 			"emulator-sender": [],
-			"emulator-receiver": []
+			"emulator-receiver": [],
+			"emulator-user1": [],
+			"emulator-user2": []
 		},
 		"testnet": {
 			"lost-and-found-testnet": [
 				"LostAndFound",
-				"FeeEstimator"
+				"FeeEstimator",
+				"LostAndFoundHelper"
```

Comment:
Looks like you're missing the mainnet addition for this contract fyi

Reply:
Thank you for the reminder! 
Just added the helper contract to mainnet!